### PR TITLE
Add install-only option for using goreleaser in user scripts

### LINKED ISSUE #250: Install only option
Thanks for a great project!

Would it be possible to add an "install only" option so that gorelease could be used in the following steps?
In our project, we use a script in our release pipeline and I had hoped to use this action to install goreleaser before running our own script.

```yaml
-
  name: Setup GoReleaser
  uses: goreleaser/goreleaser-action@v2
  with:
    version: v0.145.0
    install-only: true
```

The expected outcome of using something like the above would be that goreleaser would be installed and added to the path, allowing goreleaser to be called from scripts in a pipeline.

Let me know if you need additional information.

### PR DESCRIPTION
**Pull request by arunvelsriram**
Closes #250 

Added `install-only` option that installs GoReleaser, add it to `PATH` and exits. In the subsequent steps users can consume `goreleaser` CLI.

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -2,13 +2,22 @@ import * as git from './git';
 import * as installer from './installer';
 import * as core from '@actions/core';
 import * as exec from '@actions/exec';
+import {dirname} from 'path';
 
 async function run(): Promise<void> {
   try {
     const version = core.getInput('version') || 'latest';
-    const args = core.getInput('args', {required: true});
+    const isInstallOnly = core.getInput('install-only') === 'true';
```
Comment (crazy-max) [inline src/main.ts:10]:
```suggestion
    const isInstallOnly = /true/i.test(core.getInput('install-only'));
```

**Review by crazy-max — CHANGES_REQUESTED** [review]
Thanks for your contrib. Some small changes and also please update the README.

File: src/main.ts (hunk 1 above)
Comment (arunvelsriram) [reply src/main.ts:10]: regex `/true/i` is matching any text containing case-insensitive "true" in it.

```
> /true/i.test("text-containing-true-matches")
true
```

May be we will have to use `/^true$/i`

```
> /^true$/i.test("text-containing-true-matches")
false
```

Comment (crazy-max) [reply src/main.ts:10]: @arunvelsriram LGTM

File: README.md [hunk 2]
```diff
@@ -131,6 +146,7 @@ Following inputs can be used as `step.with` keys
 | `version`**¹**   | String  | `latest`  | GoReleaser version                        |
 | `args`           | String  |           | Arguments to pass to GoReleaser           |
 | `workdir`        | String  | `.`       | Working directory (below repository root) |
+| `install-only`   | String  | `false`   | Just install GoReleaser                   |
```
Comment (crazy-max) [inline README.md:149]:
```suggestion
| `install-only`   | Bool    | `false`   | Just install GoReleaser                   |
```

File: action.yml [hunk 3]
```diff
@@ -11,9 +11,13 @@ inputs:
     description: 'GoReleaser version'
     default: 'latest'
     required: false
+  install-only:
+    description: 'Just install GoReleaser'
+    default: false
```
Comment (crazy-max) [inline action.yml:16]:
```suggestion
    default: 'false'
```

[`default`](https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault) metadata must be a string here.

**Review by crazy-max — CHANGES_REQUESTED** [review]

**Review by crazy-max — APPROVED** [review]
LGTM thanks

**Pull request merged**